orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -468,11 +474,12 @@ func (m *swiftDialogMDMMigrator) renderMigration() error {
 					return err
 				}
 
+				m.frequency = 3 * time.Minute
```

Comment:
We'll have two different 3-minute timers in this case (one for the watcher and one for the migrator)? I think that makes it possible for the offline dialog to pop up twice in the same 3-minute period. Or is the main migrator loop doing its own offline check and skipping the render migration call? If it is possible, maybe that'll just have to become the UX at this late stage.

Reply:
Perfect, thanks for checking!